scheduler/config.edn

```diff
@@ -49,11 +49,14 @@
        :keystore-type "pkcs12"
        :keystore-pass "cookstore"}
  :rate-limit {:expire-minutes 120 ; Expire unused rate limit entries after 2 hours.
-              ; Keep these job-submisison values as they are for integration tests. Making them smaller can cause
-              ; spurious failures, and making them larger will cause the rate-limit integration test to skip itself.
-              :job-submission {:bucket-size 1000
+              ; Keep these job-launch and job-submission values as they are for integration tests. Making them smaller can cause
+              ; spurious failures, and making them larger will cause test_rate_limit_launching_jobs to skip itself.
+              :job-launch {:bucket-size 10000
+                           :enforce? true
+                           :tokens-replenished-per-minute 5000}
+              :job-submission {:bucket-size 1003
                                :enforce? true
-                               :tokens-replenished-per-minute 600}
+                               :tokens-replenished-per-minute 601}
```

Comment:
Why did we need to change these to 1003 and 601?

Reply:
We didn't need to. It was done to improve discoverability. To make it more easily greppable. (I like deliberately off-by-a-few magic numbers like this when I'm not sure the data flow..)

Reply:
I understand the grepability, but the downside is that this makes it seems as though the number needs to be 601 and not 600. It causes the reader to wonder why 601 is meaningful. I would vote for using "round" numbers whenever possible, and explaining more "specific" numbers with a comment when it's not.

Reply:
@scrosby Thoughts on this?

Reply:
This is a nice to have, but its (imo) not worth the cognitive load of adding a comment. I'll set it to round numbers.